authorization/authorization.py

```diff
@@ -268,18 +268,19 @@ def auth_enforced_via_enforcement_detectors(self, filters, requestResponse, andO
 
 def checkBypass(self, oldStatusCode, newStatusCode, oldContent,
                  newContent, filters, requestResponse, andOrEnforcement):
-    if oldStatusCode == newStatusCode:
-        auth_enforced = 0
-        if len(filters) > 0:
-            auth_enforced = auth_enforced_via_enforcement_detectors(self, filters, requestResponse, andOrEnforcement)
+    if len(filters) > 0:
+        auth_enforced = auth_enforced_via_enforcement_detectors(self, filters, requestResponse, andOrEnforcement)
         if auth_enforced:
             return self.ENFORCED_STR
-        elif oldContent == newContent:
+        else:
+            return self.BYPASSSED_STR
```

Comment:
I totally agree with the tent and approach, but I am not sure if this is accurate.
If filters did not validate enforcements it doesn't necessarily means that it is bypassed, what if the content is equal?